.gitignore

```diff
@@ -6,3 +6,4 @@ ui/.cache
 ui/dist
 ui/node_modules/
 cmd/costmodel/costmodel
+pkg/cloud/azureorphan_test.go
```

Comment:
Might we want to remove this? Or is there a good reason to keep it?

Reply:
I had this here so that I wasn't accidentally pushing code that has our azure secrets in it. It can be removed once I'm done working on the feature as well as that test file

Reply:
Let's remember to return here. It'd be cool to commit the tests, too, but with some disabling feature. Like... set the auth to empty strings, and only run the tests if the auth information is given? So that your work on that doesn't go to waste? Eventually, we should have nice ways to inject sensitive auth info into tests.